### PR TITLE
review comment #2 - avoiding constants in html-common

### PR DESCRIPTION
**Pull request by gramcha**
@aeng7 I made changes as per your review comments. Please check it out.

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -1451,13 +1451,14 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
...
       var availableLanguages = this.state.closedCaptionOptions.availableLanguages;
 
       //validate language is available before update and save
-      if (language && availableLanguages && _.contains(availableLanguages.languages, language)) {
+        if (language && availableLanguages && (_.contains(availableLanguages.languages, language) || language === noneLanaguage)) {
         this.state.closedCaptionOptions.language = this.state.persistentSettings.closedCaptionOptions.language = language;
-        var captionLanguage = this.state.closedCaptionOptions.enabled ? language : "";
-        var mode = this.state.closedCaptionOptions.enabled ? OO.CONSTANTS.CLOSED_CAPTIONS.HIDDEN : OO.CONSTANTS.CLOSED_CAPTIONS.DISABLED;
+        var captionLanguage = this.state.closedCaptionOptions.enabled && language !== noneLanaguage ? language : "";
+        var mode = this.state.closedCaptionOptions.enabled && language !== noneLanaguage ? OO.CONSTANTS.CLOSED_CAPTIONS.HIDDEN : OO.CONSTANTS.CLOSED_CAPTIONS.DISABLED;
```
Comment (pilievOoyala) [inline js/controller.js]: is false a valid value for mode and captionlanguage?

File: js/controller.js [hunk 2]
```diff
@@ -1451,13 +1451,14 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onChangeClosedCaptionLanguage: function(event, language) {
+      var noneLanaguage = 'none';
```
Comment (jpfranco) [inline js/controller.js]: Can this value be defined as a constant in the skin's constants.js? Also, 'lanaguage' is misspelled.

File: js/controller.js (hunk 1 above)
Comment (gramcha) [reply js/controller.js]: captionlanguage will be empty while disabling and mode will be assigned with OO.CONSTANTS.CLOSED_CAPTIONS.DISABLED when language === noneLanaguage

File: js/controller.js (hunk 2 above)
Comment (gramcha) [reply js/controller.js]: sure will do

Comment (gramcha) [reply js/controller.js]: @jpfranco I have added the constant to the constants.js. Please check it out.

**Comment by gramcha** [comment]
@aeng7 @pilievOoyala  Please approve the PR.

File: js/controller.js (hunk 2 above)
Comment (gramcha) [reply js/controller.js]: @jpfranco Please approve the PR.

File: js/controller.js (hunk 1 above)
Comment (pilievOoyala) [reply js/controller.js]: I'll merge this now, but in the future can you use () to make the order of operations clearer.

**Pull request merged**